mypy/types.py

```diff
@@ -2979,6 +2978,8 @@ def accept(self, visitor: TypeVisitor[T]) -> T:
 
     def relevant_items(self) -> list[Type]:
         """Removes NoneTypes from Unions when strict Optional checking is off."""
+        from mypy.state import state
```

Comment:
Nested imports are slow when compiled. This causes a performance regression, since this is called a lot.